.drone.star

```diff
@@ -12,13 +12,13 @@ def main(ctx):
       'onlyoffice': 'https://marketplace.owncloud.com/api/v1/apps/onlyoffice/4.1.3',
       'onlyoffice_sha': '8a4b626d4a522d81be6443bbfec42a6895f20d0ae61a2e5fea26ee96497e635e',
       'php': '7.3',
-      'behat_version': '10.4.0',
+      'behat_version': 'master',
```

Comment:
why master? 
is 10.4.0 qa tarball broken?

Reply:
Yes, 10.4.0 has Behat dependencies that are now "old" and do not work since last week when various upstream Behat-Mink... repos changed stuff and did releases. We had been forced to use `dev-master` in one of the dependencies for a long time, because we needed code that was fixed in upstream `dev-master` but the Behat-Mink people went for years without doing releases. Now they did releases, which is good - we can use those. But `dev-master` now points to a different/new/later commit, and that is broken (at least for us). So all old tarballs and tags that have Behat-Mink... composer.json with any reference to `dev-master` are broken.